docs/source-app/workflows/build_command_line_interface/cli.rst

```diff
@@ -0,0 +1,138 @@
+:orphan:
+
+######################################
+Develop a CLI without client side code
+######################################
+
+*************************
+1. Implement a simple CLI
+*************************
+
+To create your first CLI, you need to override the :class:`~lightning_app.core.flow.LightningFlow.configure_commands` hook and return a list of dictionaries where the keys are the commands and the values are the server side handlers.
+
+First, create a file ``app.py`` and copy-paste the following code in to the file:
+
+.. literalinclude:: example_command.py
+
+----
+
+**************
+2. Run the App
+**************
+
+Execute the following command in a terminal:
+
+.. code-block::
+
+    lightning run app app.py
+
+The following appears the terminal:
+
+.. code-block::
+
+    Your Lightning App is starting. This won't take long.
+    INFO: Your app has started. View it in your browser: http://127.0.0.1:7501/view
+    []
```

Comment:
```suggestion
```

Reply:
This is added on purpose. On the first execution, the self.names is empty.